src/main/java/net/minecraftforge/fml/client/ICustomSplashScreen.java

```diff
@@ -0,0 +1,14 @@
+package net.minecraftforge.fml.client;
+
+/**
+ * This class defines a replacement for the default forge splash screen.
+ *
+ * Only one splash screen can be registered at a time. It it registered with SplashProgress.setCustomSplashScreen().
+ * If you use this, make it a configuration option.
+ */
+public interface ICustomSplashScreen {
+    /**
+     * Render a single frame of the splash screen.
+     */
+    void renderFrame();
```

Comment:
I suggest adding `SplashFontRenderer` as a parameter here. It would be better than using a `public static` field.